dev_build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
Please put this script under the `script` directory, and add a comment to make it clear that it's for vscode build.

Reply:
I didn't notice the existence of the "apollo_docker.sh" file. With this file, I think my script file is unnecessary. I will change my code and submit again. Thank you for your valuable suggestions.